plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateListPage.java

```diff
@@ -275,10 +277,18 @@ private void createProjectControls(Composite parent, DataBindingContext dbc) {
 			.align(SWT.FILL, SWT.CENTER).grab(true, false)
 			.applyTo(projectsViewer.getControl());
 
+		final OpenShiftExplorerLabelProvider labelProvider = new OpenShiftExplorerLabelProvider();
 		projectsViewer.setContentProvider(new ObservableListContentProvider());
 		projectsViewer.setLabelProvider(new ObservableTreeItemLabelProvider());
 		projectsViewer.setInput(
 				BeanProperties.list(ITemplateListPageModel.PROPERTY_PROJECT_ITEMS).observe(model));
+		projectsViewer.setSorter(new ViewerSorter() {
+			@Override
+			public int compare(Viewer viewer, Object e1, Object e2) {
+				return labelProvider.getText(e1).compareTo(labelProvider.getText(e2));
```

Comment:
I think we already have a project sorter somewhere, that specifically puts default/origin projects on top. @jcantrill should know better.